INSTALL.md

```
@@ -0,0 +1,212 @@
+   cd /path/to/your/project
+   /list
+   ```
+
+## Other Platforms
+
+### Codex / OpenCode
+
+For platforms that support remote instruction fetching:
+
+1. **Fetch and follow remote instructions**:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)